src/app/main-overview/index.ts

```diff
@@ -0,0 +1,24 @@
+import { sources } from './sources'
+import type { ServiceDefinition } from '@/services/utils'
+import { token } from '@/services/utils'
+
+type Token = ReturnType<typeof token>
+type Sources = ReturnType<typeof sources>
+
+const $ = {
+  sources: token<Sources>('mesh.sources'),
```

Comment:
this should be renamed, but we don't use this value yet so it won't blow up

Reply:
Fixed. Also found a few more wrong instances.